app/dashboard/migrations/0004_alter_sensitivityanalysisgraph_y.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2 on 2023-08-17 15:15
```

Comment:
It seems to me that this should not be there
To only perform migrations on cp_nigeria,  you should do `python manage.py cp_nigeria makemigrations `